terra-app-helm/aou-rstudio-chart/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
Would we ever enable HPA in a single-user RStudio deployment?

Reply:
good point. Those were auto created by `helm create`